### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by nambrot**
Updates various dependencies as recommended by dependabot

### CODE REVIEW

File: solidity/core/package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "solhint-plugin-prettier": "^0.0.5",
     "solidity-coverage": "^0.7.14",
     "ts-generator": "^0.1.1",
-    "typechain": "8.0.0",
+    "typechain": "8.1.0",
```
Comment (jmrossy) [inline solidity/core/package.json:27]: It's better if we keep versions of important deps like this consistent across the monorepo, so if this changes consider changing solidity/app/package.json as well.
But @yorhodes mentioned he had a problem with 8.1.0 so pinned it to 8

Comment (yorhodes) [reply solidity/core/package.json:27]: https://github.com/dethcrypto/TypeChain/issues/721

Comment (nambrot) [reply solidity/core/package.json]: Gotcha

Comment (nambrot) [reply solidity/core/package.json]: Removed it for now

**Review by jmrossy — APPROVED** [review]
🚢

**Comment by jmrossy** [comment]
Superseded by #696 and some of these were optional (not security updates) that we can ignore for now

**Pull request closed without merging**